packages/@lwc/engine-core/src/framework/modules/computed-class-attr.ts

```diff
@@ -21,8 +21,7 @@ import { VBaseElement, VStaticPart } from '../vnodes';
 const classNameToClassMap = create(null);
 
 export function getMapFromClassName(className: string | undefined): Record<string, boolean> {
```

Comment:
We should update the `className` argument to include `null` here too.

Reply:
https://github.com/salesforce/lwc/issues/4104